styles.css

```diff
@@ -79,3 +80,187 @@ h1 {
   border: 2px solid;
   border-color: #1a2236;
 }
+
+.projects {
+  display: flex;
+  flex-direction: column;
+  background-color: #1a2236;
+  width: 100%;
+}
+
+h2 {
+  margin-left: 16px;
+  margin-top: 30px;
+  margin-bottom: 20px;
+  font-family: 'Playfair Display', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 50px;
+  line-height: 50px;
+  color: #1a2236;
+  padding-top: 15px;
+  padding-bottom: 10px;
+}
+
+.lap1,
+.lap2,
+.lap3,
+.lap4,
+.lap5 {
+  width: 100%;
+  margin-bottom: -9px;
+}
+
+.projects1b3 {
+  display: flex;
+  flex-direction: column;
+  background-color: #1a2236;
+  padding-bottom: 20px;
+}
+
+h3 {
+  margin-left: 16px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 20px;
+  line-height: 80%;
+  color: #fff;
+  padding-top: 5px;
+  padding-bottom: 5px;
+}
+
+.b3 {
+  display: flex;
+  padding-left: 16px;
+}
+
+.sbutton {
+  width: fit-content;
+  height: 20px;
+  background: #98dede;
+  color: #1a2236;
+  margin-right: 15px;
+  font-size: 10.3334px;
+  text-align: center;
+  border: 0;
+}
+
+.start1 {
+  margin-left: 16px;
+  width: 178px;
+  height: 44px;
+  color: #fff;
+  font-family: 'Poppins', sans-serif;
+  font-size: 16px;
+  font-weight: 500;
+  align-self: center;
+  background: #1a2236;
+  border: 1px solid #fff;
+  margin-top: 20px;
+}
+
+.start1:hover {
+  color: #1a2236;
+  background: white;
+  border: 1px solid;
+  border-color: #1a2236;
+}
+
+.start1:active {
+  color: #1a2236;
+  background: white;
+  border: 2px solid;
+  border-color: #1a2236;
+}
+
+.start2 {
+  margin-left: 16px;
+  width: 288px;
+  height: 50px;
+  color: white;
+  font-family: 'Poppins', sans-serif;
+  font-size: 16px;
+  font-weight: 500;
+  background: #1a2236;
+  background-image: url(./images/descarga.svg);
+  background-repeat: no-repeat;
+  background-position-x: 76%;
+  background-position-y: center;
+  border: 0%;
+}
```

Comment:
As suggested by the previous reviewer, kindly let the `Get my CV` button align with the about text.

One way you can achieve this is by setting the width of the button as 100% and assigning an equal margin right, margin left to both elements.

```
.start2 {
  width: 100%
  margin: 0 16px
}
```

Your design:
![image](https://user-images.githubusercontent.com/20554583/198838982-9c8c8e3f-a767-4f3e-9d72-6b07f0f6b148.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/198839013-dfb89fc6-6841-446f-ba44-4aee9b0065bc.png)

Reply:
Hi @emmiiorji, hope you re very well. I already applied the width at 100%. However, I did not work properly. So I just set the margins (right and left to 16px) and now it is responsive and it is aligned with the paragraph and the title of the section. I just want to confirm that I can submit it now.

Reply:
Hi @Juanpa8830 


Sorry, using `calc(100% - 32px)` would work better if there's margin right, margin left of 16px each

Reply:
Hi @Juanpa8830.

Kindly go ahead and submit, the next code reviewer will do justice.